test/functional-tests/specs/pageActions.spec

```diff
@@ -100,4 +100,8 @@
 * Assert text " " exists on the textArea.
    |Type   |Selector|
    |-------|--------|
-   |textBox|Username|
\ No newline at end of file
+   |textBox|Username|
+
+## Navigate within Page
+* Navigate to "https://github.com/getgauge/taiko#whats-taiko"
```

Comment:
@suganya-sangith For functional tests, we try to avoid using external urls as much as possible. 
Can you please write a unit test instead using a simple html. Thanks.

Reply:
@nehashri Since in unit-tests pageHandler is mocked, I can't assert whether the event is triggered.

Instead I could make this as static page

Reply:
Yes, static page should work too.

Reply:
I have made the changes, please have a look

Reply:
@nehashri Can you try rerunning the taiko CI, not sure why Linux node_10_x and Windows alone is failing